build.psm1

```diff
@@ -533,7 +533,7 @@ function Restore-PSPester
         [ValidateNotNullOrEmpty()]
         [string] $Destination = ([IO.Path]::Combine((Split-Path (Get-PSOptions -DefaultToNew).Output), "Modules"))
     )
-    Save-Module -Name Pester -Path $Destination -Repository PSGallery -RequiredVersion "4.4.1"
```

Comment:
I believe updating Pester version should be always another PR. We could do this after 6.2 Preview4 if we want the new version.

Reply:
Some of the tests are only failing in VSTS and not locally on my Windows development machine.  Trying to figure out the differences in environment; one of which is the version of Pester in VSTS vs what I have locally.  Was trying to see if a change in Pester contributed to some tests failing.  Since it looks like the same failures, I'll revert this.  However, I'll likely need to add more test tracing to figure this out since it doesn't fail locally.